tests/unit/test_charm.py

```diff
@@ -228,7 +234,9 @@ def test_config_changed(self):
         got_plan["services"]["superset"]["environment"][
             "ADMIN_USER"
         ] = "unique-user"
-        self.assertEqual(got_plan["services"], want_plan["services"])
+        self.assertEqual(
+            got_plan["services"]["superset"], want_plan["services"]["superset"]
```

Comment:
Maybe add another test to check that the new service exists in the list of services?